src/wolff_client.erl

```diff
@@ -147,8 +156,8 @@ safe_call(Pid, Call) ->
 recv_leader_connection(Client, Topic, Partition, Pid, MaxPartitions) ->
```

Comment:
this function is called from wolff_producer.erl in a timer:apply_after
it should accept aliased topic instead.